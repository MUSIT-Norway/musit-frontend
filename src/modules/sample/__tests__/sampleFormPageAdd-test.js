/**
 * Created by steinaol on 06.03.17.
 */
import { shallow } from 'enzyme';
import { shallowToJson } from 'enzyme-to-json';
import React from 'react';
import SampleFormPageAdd from '../SampleFormAddContainer';


describe('AnalysisSampleFormPageAdd', () => {

  it('should display correctly', () => {
    const wrapper=shallow(<SampleFormPageAdd/>);
    expect(shallowToJson(wrapper)).toMatchSnapshot();
  });
});
