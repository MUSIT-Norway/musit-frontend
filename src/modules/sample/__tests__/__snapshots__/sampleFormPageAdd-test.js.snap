exports[`AnalysisSampleFormPageAdd should display correctly 1`] = `<Injected />`;
